src/wallet/wallet.cpp

```diff
@@ -4097,7 +4097,7 @@ bool CWallet::Verify(interfaces::Chain& chain, const WalletLocation& location, b
 
 std::shared_ptr<CWallet> CWallet::CreateWalletFromFile(interfaces::Chain& chain, const WalletLocation& location, uint64_t wallet_creation_flags)
 {
-    const std::string& walletFile = location.GetName();
+    const std::string& walletFile = (location.GetName() == "" ? "[default wallet]" : location.GetName());
```

Comment:
```suggestion
    const std::string& log_wallet_name = (location.GetName() == "" ? "[default wallet]" : location.GetName());
```

nit: I'd feel more safe if this was renamed to `log_wallet_name`

Reply:
that makes sense, I will do that.